docs/editor/debugging.md

```diff
@@ -319,6 +319,27 @@ Source maps can be generated with two kinds of inlining:
 
 VS Code supports both the **inlined source maps** and the **inlined source**.
 

```

Comment:
I think using the term `launch.json` is clearer than launch config. Also I feel that launch.json is what we usually use in other places.